### PR TITLE
addpatch: python-leather 0.4.0

### PR DESCRIPTION
**Pull request by kanakshilledar**
fix checksum mismatch error.
successfully passing the tests.

### CODE REVIEW

File: python-leather/riscv64.patch [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+Signed-off-by: Kanak Shilledar <kanakshilledar111@protonmail.com>
+---
+ PKGBUILD | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/PKGBUILD b/PKGBUILD
+index 3b6d2eb..5c8d3ed 100644
+--- a/PKGBUILD
++++ b/PKGBUILD
```
Comment (Xeonacid) [inline python-leather/riscv64.patch:17]:
```suggestion
--- PKGBUILD
+++ PKGBUILD
```
Adhere to our patch format

**Review by Xeonacid — COMMENTED** [review]
Thanks for your contribution!
Could you please also report the checksum mismatch to Arch Linux?

File: python-leather/riscv64.patch (hunk 1 above)
Comment (kanakshilledar) [reply python-leather/riscv64.patch]: Done

**Comment by kanakshilledar** [comment]
Yes I have mailed them for their gitlab access

**Review by Xeonacid — APPROVED** [review]
Thanks!

**Comment by kanakshilledar** [comment]
Thanks! :)

**Pull request merged**